tests/kafkatest/services/kafka/kafka.py

```diff
@@ -451,7 +451,8 @@ def _kafka_topics_cmd(self, node, force_use_zk_connection):
         set. If Admin client is not going to be used, don't set the environment variable.
         """
         kafka_topic_script = self.path.script("kafka-topics.sh", node)
-        skip_security_settings = force_use_zk_connection or not self.all_nodes_topic_command_supports_bootstrap_server()
+        skip_security_settings = force_use_zk_connection or not self.all_nodes_topic_command_supports_bootstrap_server() \
+                                 or self.interbroker_security_protocol == SecurityConfig.PLAINTEXT
```

Comment:
should we add a method to KafkaService or SecurityConfig that tells you which listener to use for tools? Using inter-broker here looks a bit odd, even though it becomes clear when you look at the other changes.

Reply:
Yes, good point, I think it would be a helpful to consolidate the assumptions into one method (or a couple or a few, at most).  I'll see what I can do about refactoring those assumptions out and reusing the logic instead of having it sprinkled around so much as it is now.